migrations/versions/930_audit_event_data_field_non_nullable.py

```diff
@@ -0,0 +1,32 @@
+"""audit_event_data_field_non_nullable
+
+Revision ID: 930
+Revises: 920
+Create Date: 2017-06-19 11:31:56.252791
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '930'
+down_revision = '920'
+
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+def upgrade():
+    op.alter_column(
+        'audit_events',
+        'data',
+        existing_type=postgresql.JSON(astext_type=sa.Text()),
+        nullable=False,
```

Comment:
So it looks like the `nullable` flag was set on the column 6 months ago but the migration hasn't been generated until now for some reason.

Are we sure that there are no `data=NULL` or `data='null'` events in production DB?

In general, I don't see a reason why data would be required for an audit event, but if the migration will apply without issues then I guess we might as well roll it out and ask everyone to do `data={}` when they don't have anything to store.